docs/architecture/adr-015-crypto-encoding.md

```diff
@@ -21,23 +21,23 @@ representation of the pubkey.
 This has two significant drawbacks.
 Amino encoding is less space-efficient, due to requiring support for upgradability.
 Amino encoding support requires forking protobuf and adding this new interface support
-option in the langauge of choice.
+option in the language of choice.
 
 The reason for continuing to use amino however is that people can create code
 more easily in languages that already have an up to date amino library.
 It is possible that this will change in the future, if it is deemed that
-requiring amino for interacting with tendermint cryptography is unneccessary.
+requiring amino for interacting with Tendermint cryptography is unnecessary.
 
 The arguments for space efficiency here are refuted on the basis that there are
-far more egregious wastages of space in the SDK.
+far more egregious wastage's of space in the SDK.
```

Comment:
This change is incorrect.

Reply:
hm okay will revert, spell check in editor was throwing me the error

Reply:
Yeah, the plural s-form of wastage is a bit esoteric, so not surprised the spell checker isn't pleased, but it's the correct form. Could always rewrite the sentence to be a bit simpler, but 🤷‍♀️.